src/components/Input/QtySelector/QtySelector.js

```diff
@@ -101,15 +109,19 @@ class QtySelector extends Component {
         if (state.value === min) {
           return {};
         }
-        return { value: state.value - 1 };
+        if (checkValue(state.value - 1)) {
```

Comment:
Should `checkValue` be checked too if it's passed before invoking it?

Reply:
I can add in a check! `checkValue` also has a default value `() => true` though. Should I keep it?

Reply:
Oh i didn't see the default! If that's the case then it's all good!

Reply:
Awesome! Thank you!!